src/bitmessagekivy/mpybit.py

```diff
@@ -1,34 +1,198 @@
+        importlib.import_module(import_from, import_to)
+
+
+class Lang(Observable):
+    """UI Language"""
+
+    observers = []
+    lang = None
+
+    def __init__(self, defaultlang):
+        super(Lang, self).__init__()
+        self.ugettext = None
+        self.lang = defaultlang
+
+    @staticmethod
+    def _(text):
+        return text
+
+
+class NavigationItem(OneLineAvatarIconListItem):
+    """UI for NavigationItem class on Side Navigation bar"""
+    badge_text = StringProperty()
+    icon = StringProperty()
+    active = BooleanProperty(False)
+
+    def currentlyActive(self):
+        """Currenly active"""
+        for nav_obj in self.parent.children:
+            nav_obj.active = False
+        self.active = True
+
+
+class NavigationDrawerDivider(OneLineListItem):
+    """
+    A small full-width divider line for Side Navigation bar
+    """
+
+    disabled = True
+    divider = None
+    _txt_top_pad = NumericProperty(dp(8))
+    _txt_bot_pad = NumericProperty(dp(8))
 
+    def __init__(self, **kwargs):
+        # pylint: disable=bad-super-call
+        super(OneLineListItem, self).__init__(**kwargs)
+        self.height = dp(16)
 
-class NavigateApp(App):
+
+class NavigationDrawerSubheader(OneLineListItem):
+    """
+    A subheader for separating content in :class:`MDNavigationDrawer`
+    Works well alongside :class:`NavigationDrawerDivider`
+    """
+
+    disabled = True
+    divider = None
+    theme_text_color = 'Secondary'
+
+
+class ContentNavigationDrawer(BoxLayout):
+    """Helper for Side navigation bar which contains screen names"""
+
+    def __init__(self, *args, **kwargs):
+        """Method used for to initialize side navbar"""
+        super(ContentNavigationDrawer, self).__init__(*args, **kwargs)
+        Clock.schedule_once(self.init_ui, 0)
+
+    def init_ui(self, dt=0):
+        """Clock Schdule for class contentNavigationDrawer"""
+        self.ids.scroll_y.bind(scroll_y=self.check_scroll_y)
+
+    def check_scroll_y(self, instance, somethingelse):
+        """show data on scroll down"""
+        if self.ids.identity_dropdown.is_open:
+            self.ids.identity_dropdown.is_open = False
+
+
+class CustomSpinner(Spinner):
+    """
+    A Dropdown on side navigation bar which hold the identity and can switch to other identity
+    """
+
+    def __init__(self, *args, **kwargs):
+        """Method used for setting size of spinner"""
+        super(CustomSpinner, self).__init__(*args, **kwargs)
+        self.dropdown_cls.max_height = Window.size[1] / 3
+        self.values = list(addr for addr in config.addresses()
+                           if config.get(str(addr), 'enabled') == 'true')
+
+
+class NavigateApp(MDApp):
     """Navigation Layout of class"""
 
     def __init__(self):
         super(NavigateApp, self).__init__()
         self.kivy_state_obj = KivyStateVariables()
 
-    def build(self):
+    title = "PyBitmessage"
+    identity_list = ListProperty(
```

Comment:
maybe there can be a method to reload addresses? this shouldn't be a class variable